### ISSUE TITLE
Select Learning Rate Schedules

### ISSUE THREAD

**Issue by sarapapi** [ThilinaRajapakse/simpletransformers#703]
Hello
If I am not wrong, the only available learning rate schedule is `get_linear_schedule_with_warmup`, I'd like to use, for example, a constant learning rate.
Is it possible to add the scheduler type to the model args?
There are several schedulers (from Pytorch) already supported by HuggingFace library.
Thanks

**Comment by ThilinaRajapakse** [comment]
Yeah, it will be useful to have the other schedules as well. I'll add this as a feature request.
